### PR TITLE
Prepare a 0.3.12 release.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# lang_tester 0.3.12 (2020-07-13)
+
+* Failed stderr/stdout tests now use fm to show the offending line and upto 3
```
Comment (vext01) [inline CHANGES.md:3]: Typo. "upto" -> "up to".

**Comment by vext01** [comment]
Just a tiny typo.

File: CHANGES.md (hunk 1 above)
Comment (ltratt) [reply CHANGES.md]: Fixed in https://github.com/softdevteam/lang_tester/pull/69/commits/9b04ef3d6a854173ac0b7f14268e3958b600f97d.

**Comment by vext01** [comment]
Please squash

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**